pom.xml

```diff
@@ -30,6 +30,7 @@
     <application-poc-tools.version>2.3.0-SNAPSHOT</application-poc-tools.version>
     <coffee-boots.version>4.0.0</coffee-boots.version>
     <org.json.version>20250107</org.json.version>
+    <commons-collections.version>3.2.2</commons-collections.version>
```

Comment:
I compiled project without this dependency, so it can be deleted, also we have unused property commons-collections4.version, I think this property also can be deleted